### ISSUE TITLE
Buttons have unclear labels

### ISSUE THREAD

**Issue by Woebin** [JohnXLivingston/peertube-plugin-livechat#724]
**Describe the bug**
This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 2.4.6 (AA) Headings and Labels](https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels).

In the chat widget, there's a hamburger menu button associated with each message, which generally appears on mouse hover. This button has `aria-label="Menu"`, which doesn't give enough context to explain what it's really for.

![Button with aria-label="Menu"](https://github.com/user-attachments/assets/c05713a0-1af9-4a06-8749-8b8e4937683f)

Since there's a button like this for each message, and they all have the same aria-label, it can get very confusing and difficult to tell them apart. Here's VoiceOver on MacOS listing all the buttons it can find on the page:

![VoiceOver listing all buttons on the page, including multiple ones simply named "Menu".](https://github.com/user-attachments/assets/5a772366-c920-4f54-982d-516d7d403b76)

**Expected behavior**
Labels need to be clear and descriptive of their associated purpose. In this case that means specifying which chat message each such button is associated with, which may be difficult to do in a way that isn't overly verbose and cumbersome.

**Server:**
 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 14.0.0

**Desktop:**
 - OS: MacOS 15.5
 - Browser: Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).
